server/src/main/java/com/showmetheroapes/paracord/controllers/StrandController.java

```diff
@@ -0,0 +1,24 @@
+package com.showmetheroapes.paracord.controllers;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/api/v1/strands", produces = "application/json")
+public class StrandController {
+
+  /**
+   * Endpoint to get all of the currently available strands
+   *
+   * @return a list of strand names
+   */
+  @GetMapping
+  public List<String> findAll() {
+    final List<String> strands = new ArrayList<>();
+    strands.add("Test");
+    return strands;
+  }
+}
```

Comment:
Also where the heck do these controllers get attached to the application??

Reply:
So this is some automagic from Spring. The `@SpringBootApplication` in ParacordApplication causes anything under the `com.showmetheroapse.paracord` package to search for Spring annotated classes like `@RestController`.

Reply:
Wow that's cool and also terrifying lol thanks!